docs/user_guide/Certificates/Official_Certificates.inc

```diff
@@ -1,38 +1,54 @@
 Installing Official Certificates
 --------------------------------
 
-Below are the steps to install official certificates for a SIMP client on
-the SIMP server:
+This section describes how to install infrastructure certificates from an
+official certificate authority on the puppetserver for distribution to client
+servers.  You need to have simp_options::pki set to ``simp`` on the client for
+this to work.
+
+The key distribution directory on the puppetserver is located under the site_files directory.
+The site_files directory is an alternate module path set in each environment's environment.conf file.
+By default the site_files directory is ``/var/simp/environments/simp/site_files/``
+and the key distribution directory is ``pki_files/files/keydist``.  The system expects each client
+to have a directory that is its FQDN and the certs should be named with the FQDN.
```

Comment:
https://github.com/simp/pupmod-simp-pki/pull/45 allows users to override this.  Should this description reflect this new option, or would that be too much detail?